### PR TITLE
Added truffle

### PR DESCRIPTION
**Pull request by evgeth**

### CODE REVIEW

File: contracts/Association.sol [hunk 1]
```diff
@@ -116,32 +116,32 @@ contract Association is owned {
...
   event ReceivedEther(address from, uint256 amount);
-  
+
   // Constructor
   function Association(
-    AbstractToken sharesAddress,
-    uint256 debateTimeInMinutes,
-    uint256 minimumSharesToPassVote)
+    address sharesAddress,
+    uint256 _debateTimeInMinutes,
```
Comment (evgeth) [inline contracts/Association.sol:123]: Constructor arguments  should be named differently from state variables.

File: contracts/Association.sol [hunk 2]
```diff
@@ -116,32 +116,32 @@ contract Association is owned {
...
 
   // TODO: create mechanism for these changes to be altered
   // via a proposal mechanism instead of relying on owner.
   // This function will still be used, but the contract's owner will change.
   // This is how ethereum can be made dynamic.
   function changeVotingRules(
-      AbstractToken sharesAddress,
-      uint256 debateTimeInMinutes,
-      uint256 minimumSharesToPassVote)
+      address sharesAddress,
```
Comment (evgeth) [inline contracts/Association.sol:134]: This is not a Token type, just an address

File: contracts/Sale.sol [hunk 3]
```diff
@@ -71,19 +71,19 @@ contract Sale {
   event GoalReached(address beneficiary, uint amountRaised);
   event FundsReceived(address backer, uint amount);
   event FundsWithdrawal(address recipient, uint amount);
-  
-  function Crowdsale(
```
Comment (evgeth) [inline contracts/Sale.sol:75]: Name of the constructor should _match_ name of the contract :)

File: migrations/2_deploy_contracts.js [hunk 4]
```diff
@@ -3,7 +3,12 @@ var Association = artifacts.require("./Association.sol");
...
 module.exports = function(deployer) {
-  deployer.deploy(Token);
-  deployer.deploy(Association);
-  deployer.deploy(Sale);
+
+  console.log(web3.eth.accounts[0]);
+
+  deployer.deploy(Token, "Token", "TKN", 8, 10000000000000).then(function () {
+    deployer.deploy(Association, Token.address, 10080, 20000).then(function() {
+      deployer.deploy(Sale, web3.eth.accounts[0], 200, 43200, 1, Token.address);
```
Comment (evgeth) [inline migrations/2_deploy_contracts.js:11]: This is how you deploy contracts in a sequence in truffle.

**Review by evgeth — COMMENTED** [review]
Important PR, merge it before making changes to the code of the contracts.

File: contracts/Association.sol (hunk 1 above)
Comment (JoshBroomberg) [reply contracts/Association.sol:123]: Thanks

File: migrations/2_deploy_contracts.js (hunk 4 above)
Comment (JoshBroomberg) [reply migrations/2_deploy_contracts.js:11]: Dope

**Review by JoshBroomberg — APPROVED** [review]

**Pull request merged**